sdk/appcenter-auth/src/main/java/com/microsoft/appcenter/auth/Auth.java

```diff
@@ -380,20 +380,19 @@ public void run() {
 
     @WorkerThread
     private synchronized void processDownloadedConfig(String payload, String eTag) {
-        boolean continueSignIn = mAuthenticationClient == null && mLastSignInFuture != null;
```

Comment:
Incorrect change. You can't check it after `initAuthenticationClient`. Add comment about that.